web-authentication-ui/src/test/java/com/okta/webauthenticationui/ForegroundActivityTest.kt

```diff
@@ -97,26 +108,6 @@ class ForegroundActivityTest {
         verify(redirectCoordinator, never()).emitError(any())
     }
 
-    @Test fun testCancellation() {
-        val launchedFromActivity = Robolectric.buildActivity(Activity::class.java).create().get()
-        val intent = ForegroundActivity.createIntent(launchedFromActivity)
-        val controller = Robolectric.buildActivity(ForegroundActivity::class.java, intent)
-        val activity = controller.get()
-        val redirectCoordinator = mock<RedirectCoordinator> {
-            on { launchWebAuthenticationProvider(any(), any()) } doReturn true
-            onBlocking { runInitializationFunction() } doAnswer {
-                RedirectInitializationResult.Success(mock(), Any())
-            }
-        }
-        ForegroundViewModel.redirectCoordinator = redirectCoordinator
-        controller.create().resume()
-        assertThat(activity.isFinishing).isFalse()
-        controller.pause().resume()
-        assertThat(activity.isFinishing).isTrue()
```

Comment:
Deleted this test case which was mainly testing this flow. ForegroundActivity pausing, then resuming isn't necessarily cancellation of our login flow. ForegroundActivity can be paused while the browser login window is displayed, since it gets backgrounded in that case. Previously we were incorrectly calling finish() in this case, and we are no longer doing that with this PR.